src/App.tsx

```diff
@@ -0,0 +1,155 @@
+        <div className={styles.options}>
+          <div className={styles.dropdownGroup}>
+            <label htmlFor="type" className={styles.dropdownLabel}>
+              Type:
+            </label>
+            <select
+              id="type"
+              onChange={({ target }) =>
+                setType(target.value as unknown as Type)
+              }
+              value={type}
+              className={styles.dropdown}
+            >
+              <option value="default">Default</option>
+              <option value="warning">Warning</option>
+              <option value="success">Success</option>
+              <option value="error">Error</option>
+            </select>
+          </div>
+          <div className={styles.dropdownGroup}>
+            <label htmlFor="width" className={styles.dropdownLabel}>
+              Width:
+            </label>
+            <select
+              id="width"
+              onChange={({ target }) =>
+                setWidth(target.value as unknown as Width)
+              }
+              value={width}
+              className={styles.dropdown}
+            >
+              <option value="default">Default</option>
+              <option value="small">Small</option>
+              <option value="medium">Medium</option>
+              <option value="large">Large</option>
+              <option value="full">Full</option>
+            </select>
+          </div>
+          <div className={styles.dropdownGroup}>
+            <label htmlFor="position" className={styles.dropdownLabel}>
+              Position:
+            </label>
+            <select
+              id="position"
+              onChange={({ target }) =>
+                setPosition(target.value as unknown as Position)
+              }
+              value={position}
+              className={styles.dropdown}
+            >
+              <option value="default">Default</option>
+              <option value="tleft">Top Left</option>
+              <option value="tright">Top Right</option>
+              <option value="tcenter">Top Center</option>
+              <option value="bleft">Bottom Left</option>
+              <option value="bright">Bottom Right</option>
+              <option value="bcenter">Bottom Center</option>
+            </select>
+          </div>
+          <div className={styles.dropdownGroup}>
+            <label htmlFor="duration" className={styles.dropdownLabel}>
+              Duration:
+            </label>
+            <select
+              id="duration"
+              onChange={({ target }) =>
+                setTimer(target.value as unknown as Timer)
+              }
+              value={timer}
+              className={styles.dropdown}
+              aria-label="Duration in seconds"
+            >
+              <option value={1000}>1s</option>
+              <option value={2000}>2s</option>
+              <option value={3000}>3s</option>
```

Comment:
same for this as well or maybe even break the select component to become a standalone(this could be another PR).